### DISCUSSION TITLE
[LTSC 2018] explorer doesn't start after updating to 22621.1413.54.3

### DISCUSSION THREAD

**Discussion by roytam1** [valinet/ExplorerPatcher#1142] [General]
EP 22621.1344.53.1 works fine and it told me there is an EP update 22621.1413.54.3 today. I installed it and explorer won't start unless I reboot and downgrade back to 22621.1344.53.1.

**Comment by topia** [comment]
~~There are only two changes between those two versions. Could you let me know if you are using the weather feature? I suspect [this change](https://github.com/valinet/ExplorerPatcher/commit/a5e5287954410e82f5bb146b7c58474be8ee4e89) hurts LTSC 2018.~~

**Reply by roytam1** [reply]
> Could you let me know if you are using the weather feature?

no I don't, weather is always hidden even before I install EP.

**Reply by topia** [reply]
Ah, I'm sorry, you say downgraded to "1344.53.1", but I treated it as 54.1. please disregard.

**Comment by valinet** [comment]
Yeah, sorry about that, testing under all configs is a pain the back. The development model definitely has to be changed, it cannot go on like this forever, the project in its current form has outgrown the ability to be manageable.

Anyway, I fixed this in the latest version (54.5).
